.buildkite/cache-builder.yml

```diff
@@ -7,14 +7,14 @@
 common_params:
   # Common plugin settings to use with the `plugins` key.
   - &common_plugins
-    - automattic/a8c-ci-toolkit#2.15.1
+    - automattic/a8c-ci-toolkit#2.16.0
     - automattic/git-s3-cache#1.1.4:
         bucket: "a8c-repo-mirrors"
         repo: "automattic/wordpress-ios/"
   # Common environment values to use with the `env` key.
   - &common_env
     # Be sure to also update the `.xcode-version` file when updating the Xcode image/version here
-    IMAGE_ID: xcode-14.2
+    IMAGE_ID: xcode-14.3
```

Comment:
Using the `xcode-14.3.1-rc.1` image?

> [Resolved Issues](https://developer.apple.com/documentation/xcode-release-notes/xcode-14_3_1-release-notes#Resolved-Issues)
>
>    Fixed: When targeting devices running iOS 13, apps built with Xcode 14.3 and using Objective-C protocols from Swift, sometimes crash at launch. (108930834)

Reply:
Worth a shot!

Reply:
Tried in https://github.com/wordpress-mobile/WordPress-iOS/commit/6de2c8d7fd218a8e087387b993242ece8777be12 but it didn't work.